actions/ctf-run-tests-binary/action.yml

```diff
@@ -0,0 +1,154 @@
+name: ctf-run-tests-binary
+description: "Run CTF test binary"
+
+inputs:
+  artifacts_location:
+    required: false
+    description: Location of where error logs are written
+    default: ./logs
+  artifacts_name:
+    required: false
+    description: Name of the artifact to upload
+    default: test-logs
+  test_command_to_run:
+    required: true
+    description:
+      The command to run the tests by calling your binary (note that -test.json
+      is not supported)
+    # https://github.com/golang/go/issues/22996
+  cl_repo:
+    required: false
+    description: The Chainlink ecr repository to use
+    default: public.ecr.aws/z0b1w9r9/chainlink
+  cl_image_tag:
+    required: false
+    description: The chainlink image to use
+    default: develop
+  build_gauntlet_command:
+    required: false
+    description: How to build gauntlet if necessary
+    default: "false"
+  download_contract_artifacts_path:
+    required: false
+    description: Path where the contract artifacts need to be placed
+    default: "none"
+  token:
+    required: false
+    description: The GITHUB_TOKEN for the workflow
+    default: ${{ github.token }}
+  triggered_by:
+    required: true
+    description:
+      The triggered-by label for the k8s namespace, required for cleanup
+    default: ci
+  cache_restore_only:
+    required: false
+    description:
+      Only restore the cache, set to true if you want to restore and save on
+      cache hit miss
+    default: "false"
+  cache_key_id:
+    required: false
+    description: Cache go vendors unique id
+    default: go
+  aws_registries:
+    required: false
+    description: AWS registries to log into for the test if needed
+  aws_role_duration_seconds:
+    required: false
+    default: "3600"
+    description: The duration to be logged into the aws role for
+  dockerhub_username:
```

Comment:
I'm fine to merge this as it is, but going forward would be nice to standarize on using dashes for naming action inputs.

`dockerhub-username` instead of `dockerhub_username`